packages/gambit.rb

```diff
@@ -0,0 +1,34 @@
+require 'package'
+
+class Gambit < Package
+  description 'Efficient implementation of the Scheme programming language'
+  homepage 'http://gambitscheme.org/wiki/index.php/Main_Page'
+  version '4.9.3'
+  source_url 'https://github.com/gambit/gambit/archive/v4.9.3.tar.gz'
+  source_sha256 'a5e4e5c66a99b6039fa7ee3741ac80f3f6c4cff47dc9e0ff1692ae73e13751ca'
+
+  binary_url ({
+  })
+  binary_sha256 ({
+  })
+
+  def self.build
+    system './configure',
+             "--prefix=#{CREW_PREFIX}",
+             "--libdir=#{CREW_LIB_PREFIX}/gambit",
```

Comment:
Why not just `#{CREW_LIB_PREFIX}`?  This will make it easier for other programs to access the libraries.

Reply:
Every other packager uses this path or `/gambit-c` so I did the same.

Also, I used the FreeBSD port convention for naming the interpreter and compiler commands because one of them defaults to the same name as ghostscript's binary.